torch/quantization/fx/quantize.py

```diff
@@ -905,6 +905,11 @@ def insert_quantize_node(node: Node) -> None:
                 else:
                     env[node.name] = \
                         self.quantized_graph.node_copy(node, load_non_quantized)
+            elif node.op == 'call_function' and node.target is getattr:
+                # getattr nodes can work on quantized or non-quantized
+                # tensors
+                env[node.name] = \
+                    self.quantized_graph.node_copy(node, load_x)
```

Comment:
actually this is the definition of CopyNode.. maybe add this to https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/quantization_patterns.py#L989?